includes/class-sensei-course-progress-widget.php

```diff
@@ -191,15 +194,15 @@ function widget( $args, $instance ) {
                 $lesson_quiz_id = Sensei()->lesson->lesson_quizzes( $lesson_id );
 
 				// add 'current' class on the current lesson/quiz
-				if( $lesson_id == $post->ID || $lesson_quiz_id == $post->ID ) {
+				if( $lesson_id === $post->ID || $lesson_quiz_id === $post->ID ) {
```

Comment:
Since we are doing strict equality now, we should make sure that lesson_id is an int by casting it (i.e. `'100' === 100` will be false)